components/calendar/benches/date.rs

```diff
@@ -5,7 +5,27 @@
 mod fixtures;
 
 use criterion::{black_box, criterion_group, criterion_main, Criterion};
-use icu_calendar::{Date, DateDuration};
+use icu_calendar::{AsCalendar, Date, DateDuration};
+
+fn run_calendar_tests<A: AsCalendar>(dates: Vec<&mut Date<A>>) {
+    for date in dates {
+        // Arithmetic. black_box used to avoid compiler optimization.
+        date.add(DateDuration::new(
+            black_box(1),
+            black_box(2),
+            black_box(3),
+            black_box(4),
+        ));
+
+        // Retrieving vals
+        let _ = date.year().number;
```

Comment:
nit: these values need to be passed to `black_box` to ensure the getters aren't elided